Description: Commands in Makefile should be visible while execution
 This patch removes @ from the Makefile so that command execution is
 visible to every one.
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Last-Update: 2012-02-22
Forwarded: not-needed
--- a/Makefile
+++ b/Makefile
@@ -16,46 +16,46 @@ options:
 
 .c.o:
 	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.h config.mk
 
 config.h:
 	@echo creating $@ from config.def.h
-	@cp config.def.h $@
+	cp config.def.h $@
 
 surf: ${OBJ}
 	@echo CC -o $@
-	@${CC} -o $@ surf.o ${LDFLAGS}
+	${CC} -o $@ surf.o ${LDFLAGS}
 
 clean:
 	@echo cleaning
-	@rm -f surf ${OBJ} surf-${VERSION}.tar.gz
+	rm -f surf ${OBJ} surf-${VERSION}.tar.gz
 
 dist: clean
 	@echo creating dist tarball
-	@mkdir -p surf-${VERSION}
-	@cp -R LICENSE Makefile config.mk config.def.h README \
+	mkdir -p surf-${VERSION}
+	cp -R LICENSE Makefile config.mk config.def.h README \
 		surf-open.sh arg.h TODO.md surf.png \
 		surf.1 ${SRC} surf-${VERSION}
-	@tar -cf surf-${VERSION}.tar surf-${VERSION}
-	@gzip surf-${VERSION}.tar
-	@rm -rf surf-${VERSION}
+	tar -cf surf-${VERSION}.tar surf-${VERSION}
+	gzip surf-${VERSION}.tar
+	rm -rf surf-${VERSION}
 
 install: all
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f surf ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/surf
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f surf ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/surf
 	@echo installing manual page to ${DESTDIR}${MANPREFIX}/man1
-	@mkdir -p ${DESTDIR}${MANPREFIX}/man1
-	@sed "s/VERSION/${VERSION}/g" < surf.1 > ${DESTDIR}${MANPREFIX}/man1/surf.1
-	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/surf.1
+	mkdir -p ${DESTDIR}${MANPREFIX}/man1
+	sed "s/VERSION/${VERSION}/g" < surf.1 > ${DESTDIR}${MANPREFIX}/man1/surf.1
+	chmod 644 ${DESTDIR}${MANPREFIX}/man1/surf.1
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/surf
+	rm -f ${DESTDIR}${PREFIX}/bin/surf
 	@echo removing manual page from ${DESTDIR}${MANPREFIX}/man1
-	@rm -f ${DESTDIR}${MANPREFIX}/man1/surf.1
+	rm -f ${DESTDIR}${MANPREFIX}/man1/surf.1
 
 .PHONY: all options clean dist install uninstall
